src/utils/nodeinfo.ts

```diff
@@ -15,7 +15,15 @@ export enum NodeType {
   NODE_GREENLIGHT = 'node_greenlight',
 }
 
-export function proxynodeinfo(pk: string): Promise<Object> {
+export function proxynodeinfo(pk: string): Promise<{
+  pubkey: string
+  number_channels: number
+  open_channel_data: interfaces.Channel[]
+  largest_local_balance: number
+  largest_remote_balance: number
+  total_local_balance: number
+  node_type: NodeType.NODE_VIRTUAL
+}> {
```

Comment:
was giving an error for this type this is also one of the few odd ones

Reply:
https://github.com/kevkevinpal/sphinx-relay/pull/14 @kevkevinpal